BUILD

```diff
@@ -0,0 +1 @@
+exports_files(["root", "README.fixed", "kernel", "initrd"])
```

Comment:
I think this would look nicer as

```
exports_files([
    "root",
    "README.fixed",
    "kernel",
    "initrd",
])
```

which would also make future changes, though unlikely, less invasive.

We could also add a short comment at the top, perhaps something along the lines of

```
# Make it possible for the tarball to be consumed by http_archive()
# The files listed here will be accessible by name
```